src/main/java/org/folio/rest/impl/HoldHelpers.java

```diff
@@ -97,7 +97,7 @@ static JsonObject createCancelRequest(JsonObject body, Hold entity) {
       .put("holdingsRecordId", body.getString(JSON_FIELD_HOLDINGS_RECORD_ID))
       .put("itemId", body.getString(JSON_FIELD_ITEM_ID))
       .put("item", itemJson)
-      .put("fulfilmentPreference", body.getString(JSON_FIELD_FULFILMENT_PREFERENCE))
+      .put("fulfillmentPreference", body.getString(JSON_FIELD_FULFILLMENT_PREFERENCE))
```

Comment:
This is a compatibility breaking change and should be reflected in the major version of the module